kythe/web/site/getting-started-macos.md

```diff
@@ -66,11 +66,17 @@ done
 # Bazel.
 # See also:  https://docs.bazel.build/versions/master/install-os-x.html
 #
-# DO NOT run "brew install bazel"; the version from core does not work.
-# If you did so by mistake, run "brew uninstall bazel" first.
-brew tap bazelbuild/tap
-brew tap-pin bazelbuild/tap
-brew install bazelbuild/tap/bazel
+# Since July 2020, Bazel is now part of Homebrew Core, and can therefore
+# be installed as any ordinary Homebrew package. Note, however, that
+# version updates in Homebrew may be out of sync with Kythe versions, so
+# we recommend you manually pin the current version after installing.
+#
+# When the Kythe .bazelversion file updates, you can upgrade Bazel thus:
+#
+#   brew unpin bazel ; brew upgrade bazel && brew pin bazel
+#
+brew install bazel
```

Comment:
Is there a reason to not use [bazelisk](https://github.com/bazelbuild/bazelisk) so the user doesn't have to manually manage bazel upgrades?

Reply:
I don't know, I've never used it. I will take a look and report back. :)

Reply:
It looks like `bazelisk` works fine—at least to the point where I can build and run tests. I'll update this change to point there instead.

Reply:
great, thanks!

Reply:
I updated the text, PTAL.